crates/op-test-vectors/Cargo.toml

```
@@ -0,0 +1,21 @@
+[package]
+name = "op-test-vectors"
+description = "Types for test vectors"
+
+license = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+version = { workspace = true }
+authors = { workspace = true }
+edition = { workspace = true }
+
+[dependencies]
+op-alloy-rpc-types = { workspace = true }
+op-alloy-consensus = { workspace = true }
+alloy = { workspace = true }
+serde = { workspace = true }
+anvil-core = { workspace = true }
+
+[dev-dependencies]
+color-eyre = { workspace = true }
+serde_json = { workspace = true }
```

We can just shorthand these workspace imports.

```suggestion
license.workspace = true
repository.workspace = true
homepage.workspace = true
version.workspace = true
authors.workspace = true
edition.workspace = true

[dependencies]
op-alloy-rpc-types.workspace = true
op-alloy-consensus.workspace = true
alloy.workspace = true
serde.workspace = true
anvil-core.workspace = true

[dev-dependencies]
color-eyre.workspace = true
serde_json.workspace = true
```